app/assets/javascripts/show-hide-content.js

```diff
@@ -0,0 +1,172 @@
+;(function (global) {
```

Comment:
What's the difference between this and https://github.com/alphagov/govuk_frontend_toolkit/blob/c88df4977fbb1d63950e8d19cc7dfbd02ede0709/javascripts/govuk/show-hide-content.js ?